src/scripts/DataLayerItem.js

```diff
@@ -96,6 +183,132 @@ class Item {
   get index() {
     return this._index;
   };
+
+  /**
+   * Indicates whether the item is a data item.
+   *
+   * @returns {Boolean} true if the item is a data item, false otherwise.
+   */
+  get isData() {
+    return this._type === itemType.DATA;
```

Comment:
I think we could just export the `itemType` constants and do these checks outside rather than cluttering the item API.

`item.type === itemType.DATA` is a fairly simple check.